build.xml

```diff
@@ -44,6 +44,7 @@
             patches/7548_source-1.8.diff
             patches/7583_source-1.8.diff
             patches/7621.diff 
+            patches/7610.diff
```

Comment:
While applying patches we generally try to maintain the patches in ascending order of PR numbers, so if possible can you please place this patch before `7621.diff`, it wouldn't make any difference in this case so it's up to you.

Reply:
Ok. Thanks @Achal1607 for letting me know. I assumed that it was in ascending order of PR approval. I'll update the file. Thanks.

Reply:
Done.